lte/gateway/python/magma/enodebd/devices/BUILD.bazel

```diff
@@ -35,6 +35,7 @@ py_library(
         ":baicells",
         ":baicells_qrtb",
         ":freedomfi_one",
+        "//dp/protos:cbsd_python_proto",
```

Comment:
This does not seem to be required, i.e. `bazel build lte/gateway/python/magma/enodebd/devices:device_map` works without this line and none of the source files imports it. Can we remove it?

Reply:
Removed.